com.valvesoftware.Steam.CompatibilityTool.Proton.yml

```
@@ -94,6 +94,35 @@ modules:
       - *proton_source
 
 
+  - name: vkd3d-d3d12
+    subdir: vkd3d
+    build-options:
+      arch:
+        x86_64:
+          config-opts:
+            - WIDL=x86_64-w64-mingw32-widl
```

Nitpick: this probably should be set as an env var.
```suggestion
    build-options:
      arch:
        x86_64:
          env:
            WIDL: x86_64-w64-mingw32-widl
```
Same for `vkd3d-d3d12-32bit` module below.